src/core_plugins/kibana/public/visualize/editor/panels/save.html

```diff
@@ -19,6 +19,6 @@
     type="submit"
     class="btn btn-primary"
   >
-    Save
+    {{ opts.isAddToDashMode() ? 'Save and Return' : 'Save'}}
```

Comment:
Can we make this a little more explicit by changing it to 'Save and Add to Dashboard'?